website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -95,14 +95,16 @@ yarn trigger-react-native-release --to-version 0.69.0-rc.0 --token <YOUR_CIRCLE_
 
 To generate the changelog, we rely on a dedicated tool called [`@rnx-kit/rn-changelog-generator`](https://github.com/microsoft/rnx-kit/tree/main/incubator/rn-changelog-generator) that will parse the custom changelog messages that contributors write in their PRs.
 
+Note: Due to the number of commits for an RC0, this operation is likely to hit GitHub API limits. The snippet below set the optional `--token` arg, which can be obtained from [GitHub Settings > Developer Settings > Personal access tokens](https://github.com/settings/tokens?type=beta).
```

Comment:
You can make this a little nicer if you want to add a creation link instead: https://github.com/microsoft/rnx-kit/commit/f4809ea2fc5726cd3992f4020f87184e82f20950`

Reply:
Nice! Done :)